### PR TITLE
[MM-T1282] Add e2e test coverage for default behaviour when an empty site name is configured

### PR DESCRIPTION
**Pull request by jayaddison-collabora**
#### Summary
Adds an E2E test to cover the behaviour of the 'Site Name' customization configuration option (and default value).

#### Ticket Link
https://automation-test-cases.vercel.app/test/MM-T1282

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/system_console/site_configuration/customization_spec.js [hunk 1]
```diff
@@ -300,3 +354,18 @@ function saveSetting() {
         should('be.disabled').
         wait(TIMEOUTS.HALF_SEC);
 }
+
+function verifySiteNameInAboutModal(siteName) {
+
+    // # Open the hamburger menu
+    cy.get('button > span[class="menu-icon"]').click();
+
+    // # click to open about modal
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/site_configuration/customization_spec.js:363]:
```suggestion
    // # Click to open about modal
```

**Review by saturninoabril — APPROVED** [review]
Thank you @jayaddison-collabora! Tested and passed. LGTM, except for one request.

File: e2e/cypress/integration/system_console/site_configuration/customization_spec.js (hunk 1 above)
Comment (jayaddison-collabora) [reply e2e/cypress/integration/system_console/site_configuration/customization_spec.js:363]: Thank you!

**Review by hmhealey — APPROVED** [review]

**Review by furqanmlk — APPROVED** [review]
Thanks @jayaddison-collabora 
image

**Comment by jayaddison-collabora** [comment]
Thanks all!
